.github/workflows/dependabot-ok-to-test.yaml

```diff
@@ -0,0 +1,16 @@
+name: Add ok-to-test label for dependabot PRs
+on: pull_request
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
```

Comment:
The metadata is not used anywhere.

Reply:
I see now, it's explained here: https://github.com/dependabot/fetch-metadata?tab=readme-ov-file#auto-approving